New Appliance: ZoneMinder

Completed work on the ZoneMinder Application (no artwork yet). Repo is here https://github.com/MPTMG/ZoneMinder. Ready to be built and tested. Got a question for the experts here though.

Question 1:
Application requires timezone to be set in  /etc/php5/apache2/php.ini
During the initial setup (inithooks phase) will cat /etc/timezone reflect the timezone of the installation? If so I can script it to place the correct/current timezone in /etc/php5/apache2/php.ini. I can see UTC potentially being used though, so should I prompt for the timezone during first boot? This would require the user to enter a correctly formatted time zone "America/New_York". Let me know how you want this handled, it is easy to script either way.

Getting a build error after cloning current repo.

W: Failed to fetch http://httpredir.debian.org/debian/dists/jessie-backports/main/binary-amd64/Packages  Hash Sum mismatch

E: Some index files failed to download. They have been ignored, or old ones used instead.
/usr/share/fab/product.mk:476: recipe for target 'build/stamps/root.patched' failed
make: *** [build/stamps/root.patched] Error 100

--- a/README.rst
+++ b/README.rst
@@ -19,5 +19,6 @@ top of that:
 Credentials *(passwords set at first boot)*
 -------------------------------------------
 
--  Webmin, SSH, MySQL, Adminer: username **root**
+-  Webmin, SSH, MySQL: username **root**
+-  Adminer: username **adminer**
 -  ZoneMinder credentials set during configuration

--- a/changelog
+++ b/changelog
@@ -1,12 +1,12 @@
-turnkey-zoneminder-14.2 (1) turnkey; urgency=low
+turnkey-zoneminder-15.0 (1) turnkey; urgency=low
 
   * ZoneMinder:
 
     - Initial release of Zoneminder appliance, modified from LAMP.
-
-    - Installed ZoneMinder from Debian backports repository (v1.29.0).
+    - Installed ZoneMinder from Debian backports repository (v1.30.4).
+    [ unreleased v14.2: Michael Gurevich <mg@tapbop.com> ]
 
   * Note: Please refer to turnkey-core's changelog for changes common to all
     appliances. Here we only describe changes specific to this appliance.
 
- -- Michael Gurevich <mg@tapbop.com>  Sun, 07 May 2017 14:22:31 -0500
+ -- Jeremy Davis <jeremy@turnkeylinux.org>  Tue, 10 Jul 2018 15:43:48 +1000

--- a/overlay/etc/apt/preferences.d/zoneminder-backports.pref
+++ b/overlay/etc/apt/preferences.d/zoneminder-backports.pref
@@ -1,4 +1,4 @@
 Explanation: Pin ZoneMinder from backports
 Package: zoneminder
-Pin: release a=jessie-backports
+Pin: release a=stretch-backports
 Pin-Priority: 500

--- a/overlay/etc/apt/sources.list.d/backports.sources.list
+++ b/overlay/etc/apt/sources.list.d/backports.sources.list
@@ -1,2 +1,2 @@
-# Jessie-backports - for Adminer v4.x & ZoneMinder
-deb http://httpredir.debian.org/debian jessie-backports main
+# Stretch-backports - for ZoneMinder
+deb http://httpredir.debian.org/debian stretch-backports main

--- a/plan/main
+++ b/plan/main
@@ -7,19 +7,19 @@ python-mysqldb           /* A Python interface to MySQL                         
 libapache2-mod-perl2     /* Integration of perl with the Apache2 web server      */
 libdbd-mysql-perl        /* A Perl5 database interface to the MySQL database     */
 
-php5-xdebug
+php-xdebug
 php-pear
-php5-gd
-
+php-gd
 
 /* Zoneminder Dependencies */
+
 dbus
 javascript-common
 libarchive-zip-perl
-libavcodec56
-libavformat56
-libavresample2
-libavutil54
+libavcodec57
+libavformat57
+libavresample3
+libavutil55
 libb-hooks-endofscope-perl
 libcap-ng0
 libcommon-sense-perl
@@ -48,7 +48,7 @@ libnamespace-clean-perl
 libnet-sftp-foreign-perl
 libnet-smtp-ssl-perl
 libogg0
-libopenjpeg5
+libopenjp2-7
 libopus0
 liborc-0.4-0
 libpackage-stash-perl
@@ -59,13 +59,12 @@ libpolkit-agent-1-0
 libpolkit-backend-1-0
 libpolkit-gobject-1-0
 libproxy-tools
-libproxy1
-libschroedinger-1.0-0
+libproxy1v5
 libspeex1
 libsub-exporter-progressive-perl
 libsub-identify-perl
 libsub-name-perl
-libswscale3
+libswscale4
 libsys-mmap-perl
 libtheora0
 libtry-tiny-perl
@@ -76,8 +75,8 @@ libvlc5
 libvlccore8
 libvorbis0a
 libvorbisenc2
-libvpx1
-libx264-142
+libvpx4
+libx264-148
 libxml-libxml-perl
 libxml-namespacesupport-perl
 libxml-sax-base-perl

--- a/plan/main
+++ b/plan/main
@@ -1,87 +1,89 @@
 #include <turnkey/base>
 #include <turnkey/lamp>
 
-libapache2-mod-python    # Apache 2 module that embeds Python within the server
-python-mysqldb           # A Python interface to MySQL
+libapache2-mod-python    /* Apache 2 module that embeds Python within the server */
+python-mysqldb           /* A Python interface to MySQL                          */
 
-libapache2-mod-perl2     # Integration of perl with the Apache2 web server
-libdbd-mysql-perl        # A Perl5 database interface to the MySQL database
+libapache2-mod-perl2     /* Integration of perl with the Apache2 web server      */
+libdbd-mysql-perl        /* A Perl5 database interface to the MySQL database     */
 
 php5-xdebug
 php-pear
-php5-gd			
+php5-gd
 
-dbus			#Zoneminder Dependencies 
-javascript-common 
-libarchive-zip-perl 
-libavcodec56 
+
+/* Zoneminder Dependencies */
+dbus
+javascript-common
+libarchive-zip-perl
+libavcodec56
 libavformat56
-libavresample2 
-libavutil54 
-libb-hooks-endofscope-perl 
+libavresample2
+libavutil54
+libb-hooks-endofscope-perl
 libcap-ng0
-libcommon-sense-perl 
-libconvert-binhex-perl 
-libdate-manip-perl 
+libcommon-sense-perl
+libconvert-binhex-perl
+libdate-manip-perl
 libdbus-1-3
-libdevice-serialport-perl 
-libemail-date-format-perl 
-libglib2.0-0 
+libdevice-serialport-perl
+libemail-date-format-perl
+libglib2.0-0
 libgsm1
-libimage-base-bundle-perl 
-libimage-info-perl 
+libimage-base-bundle-perl
+libimage-info-perl
 libio-string-perl
-libjpeg62-turbo 
-libjs-jquery 
-libjs-mootools 
-libjson-any-perl 
+libjpeg62-turbo
+libjs-jquery
+libjs-mootools
+libjson-any-perl
 libjson-xs-perl
-libmailtools-perl 
-libmime-lite-perl 
+libmailtools-perl
+libmime-lite-perl
 libmime-tools-perl
-libmodule-implementation-perl 
-libmodule-runtime-perl 
+libmodule-implementation-perl
+libmodule-runtime-perl
 libmp3lame0
-libnamespace-clean-perl 
-libnet-sftp-foreign-perl 
+libnamespace-clean-perl
+libnet-sftp-foreign-perl
 libnet-smtp-ssl-perl
-libogg0 
-libopenjpeg5 
-libopus0 
-liborc-0.4-0 
+libogg0
+libopenjpeg5
+libopus0
+liborc-0.4-0
 libpackage-stash-perl
-libpam-systemd 
-libparams-classify-perl 
+libpam-systemd
+libparams-classify-perl
 libphp-serialization-perl
-libpolkit-agent-1-0 
-libpolkit-backend-1-0 
+libpolkit-agent-1-0
+libpolkit-backend-1-0
 libpolkit-gobject-1-0
-libproxy-tools 
-libproxy1 
-libschroedinger-1.0-0 
+libproxy-tools
+libproxy1
+libschroedinger-1.0-0
 libspeex1
-libsub-exporter-progressive-perl 
-libsub-identify-perl 
+libsub-exporter-progressive-perl
+libsub-identify-perl
 libsub-name-perl
-libswscale3 
-libsys-mmap-perl 
-libtheora0 
-libtry-tiny-perl 
+libswscale3
+libsys-mmap-perl
+libtheora0
+libtry-tiny-perl
 liburi-encode-perl
-libva1 
-libvariable-magic-perl 
-libvlc5 
-libvlccore8 
-libvorbis0a 
+libva1
+libvariable-magic-perl
+libvlc5
+libvlccore8
+libvorbis0a
 libvorbisenc2
-libvpx1 
-libx264-142 
-libxml-libxml-perl 
+libvpx1
+libx264-142
+libxml-libxml-perl
 libxml-namespacesupport-perl
-libxml-sax-base-perl 
-libxml-sax-perl 
-libxml-simple-perl 
+libxml-sax-base-perl
+libxml-sax-perl
+libxml-simple-perl
 libxvidcore4
-policykit-1 
-vlc-data 
+policykit-1
+vlc-data
 zip

